README.md

```diff
@@ -256,6 +256,11 @@ It is recommended to use images from public registries or pre-pull the images in
 ### Trunk port
 WMCO does not support adding Windows nodes to a cluster through a trunk port. The only supported networking setup for adding Windows nodes is through an access port carrying the VLAN traffic.
 
+## Running Windows workloads
+Be sure to set the [OS field in the Pod spec](https://kubernetes.io/docs/concepts/workloads/pods/#pod-os) to Windows
+when deploying Windows workloads. This field is used by kubelet to validate a node's operating system compatibility
+before a pod is run, as well as when enforcing OS-specific pod security standards.
```

Comment:
I suggest being more generic and saying `This fields is used to authoritatively identify the pod OS and for validation`. In addition you can saying `In OpenShift, it is used when when enforcing OS-specific pod security standards.`